traits-stubs/traits_stubs_tests/examples/Any.py

```diff
@@ -52,9 +52,8 @@ class Foo:
 
 class Superclass(HasTraits):
     x = Any()
-    # y = Any()
 
 
 class Subclass(Superclass):
-    x = Instance(Foo)
+    x = Instance(Foo)  # E: assignment
```

Comment:
This line didn't raise before this PR, and does now; I believe that this was a bug in the test. `mypy` infers from `Superclass` that the attribute `x` should have type `Any`, and complains about the non-matching assignment in the subclass, which seems like a legitimate complaint.